test/dependencies.jl

```diff
@@ -16,6 +16,11 @@ function with_temp_project(f::Function)
     end
 end
 
+# We need to be able to install stdlib JLLs
+if VERSION ≥ v"1.8.0"
+    Pkg.respect_sysimage_versions(false)
+end
```

Comment:
We may have to do this for the whole package (or for example locally only when installing the packages), instead of just when running the tests.

Reply:
It was already there, just not for Julia v1.8: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/b7b4ae9e9e7aaedc554486e7ff69df16bad532d1/src/Prefix.jl#L545-L557 :facepalm: